### ISSUE TITLE
Quote tweet by api.fxtwitter.com was crashed

### ISSUE THREAD

**Issue by Programistich** [FixTweet/FixTweet#344]
Try run https://api.fxtwitter.com/elonmusk/status/1690040807208759305 and see 500 code

**Comment by jslindgit** [comment]
Now it just gives 404.

**Comment by jslindgit** [comment]
Now it just works.

**Comment by dangeredwolf** [comment]
Twitter’s seemingly making some unexplained changes to their graphql API that I haven’t fully analyzed but the error it’s hitting is that it’s not able to read user_results somehow and it’s causing functionality to break / start working again despite not changing anything on our end. I’ll look into it since I can confirm it’s broken again.

**Comment by dangeredwolf** [comment]
It also only affects some tweets and not others, it’s fairly bizarre. I’ll try to figure out what’s happening today, hopefully figuring out what’s breaking will make more sense.

**Comment by dangeredwolf** [comment]
For some reason Twitter sometimes wraps its results in a `result` object now, but only for some tweets. Random change from what we're used to, but not a big deal, this was actually pretty easy to work around, and now the Tweet loads correctly. I'll go ahead and close this issue now, but if somehow it appears again, feel free to reopen.

**Comment by jslindgit** [comment]
Elon did it again - 404 is back.

**Comment by dangeredwolf** [comment]
I haven't noticed mass errors in Sentry currently like before, and the API URL posted above is sitll working, so if it is still broken I cannot reproduce. Maybe you were encountering temporary error? Or a cached response? If you can still recreate the bug using a different URL, please feel free to follow up.

**Comment by jslindgit** [comment]
Indeed, it's working again now. 3 hours ago I tried with many different urls, many times, and it always resulted in 404 - but these temporary errors have indeed been commong with very api recently. Let's hope that Elon settles down soon.
